draft-packages/modal/KaizenDraft/Modal/Presets/ContextModal.scss

```diff
@@ -0,0 +1,86 @@
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/layout";
+@import "~@kaizen/design-tokens/sass/spacing";
+
+.modal {
+  composes: genericModal from "../Primitives/GenericModal.scss";
+  min-width: 300px;
+  max-width: 840px;
+
+  @media (min-width: $layout-breakpoints-medium - 1px) {
+    width: 90%;
+  }
+}
+
+.header {
+  color: $color-purple-800;
+  text-align: left;
+  &.padded {
+    padding: $spacing-md $spacing-lg;
+    @media (max-width: $layout-breakpoints-medium - 1px) {
```

Comment:
Here I know why there is a `- 1px`, as the media query is only meant to apply to a screen width up to 767px, but from the iPad 768px look different.

I like the "mobile first" approach for CSS more. You essentially start with the CSS for the smallest viewport and then only use `min-width` queries from there. In this file it would also mean that the `- 1px`s would go away (which I think look a bit odd and confusing).

Happy to explain in more detail, if you want to refactor it.